Userland/Libraries/LibC/pthread.cpp

```diff
@@ -553,6 +553,12 @@ int pthread_cancel(pthread_t thread)
     return pthread_kill(thread, SIGCANCEL);
 }
 
+// https://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_testcancel.html
+void pthread_testcancel(void)
+{
+    __pthread_maybe_cancel();
```

Comment:
Hm. Looking at this more closely, I think that this first commit should be split into 3.

`LibC: Implement pthread_testcancel`

`LibC: Add missing definitions for IPv6 packet info`

and

`Ports: Build Qt modules Network and Concurrent`

Reply:
Oh sure, that sounds better, thank you very much!